tests/cmds_test.py

```diff
@@ -89,6 +88,7 @@ def f(self):
                 f.write(actual_output)
                 f.close()
         self.assertEqual(expected_output, actual_output)
+
```

Comment:
?

Reply:
`black`?

Reply:
I wonder if that was me or black/flake ...